src/libs/ajax/ajax-common.test.ts

```diff
@@ -20,6 +22,13 @@ jest.mock('src/libs/state', (): StateExports => {
   };
 });
 
+type OidcExports = typeof import('oidc-client-ts');
+jest.mock('oidc-client-ts', (): OidcExports => {
+  return {
+    ...jest.requireActual('oidc-client-ts'),
```

Comment:
Preferably, `jest.requireActual` should have a type parameter

```suggestion
    ...jest.requireActual<OidcExports>('oidc-client-ts'),
```

However, since this is returning the original module with no modifications, there's no need to mock this module at all.